### PR TITLE
fixed descriptives matrix plot

### PR DESCRIPTION
**Pull request by vandenman**
@JorisGoosen This should fix the issues with the plots.

Note that the matrix plots are a special kind of plot objects (a composition of ggplot objects).
They need their own print function, hence the changes in `writeImage.R` and `common.R`

### CODE REVIEW

**Comment by AlexanderLyNL** [comment]
The fix is to convert the matrix plot to a ggplot, isn't it? If the background is a problem, would setEPS() in R fix this then?

**Comment by vandenman** [comment]
Well since jaspResults cannot serialize any base plot, the current fix is to convert all of the base plots in descriptives to ggplots (basically implement Koen's work for descriptives only). The matrix plot is somewhat of a special case (it needs its own plot function, `gridExtra::grid.arrange()`. The current PR should fix both the issue Joris had as well as the one you showed me today.

**Comment by AlexanderLyNL** [comment]
On my mac, the binomial as factor 1. doesn't show up and 2 crashes whenever I save as eps. Let's have a look at it tomorrow.

**Comment by vandenman** [comment]
@AlexanderLyNL @JorisGoosen I think I solved the plotting, let's discuss this somewhere today.

File: JASP-R-Interface/jaspResults/R/writeImage.R [hunk 1]
```diff
@@ -28,7 +28,7 @@ writeImageJaspResults <- function(width=320, height=320, plot, obj = TRUE, relat
   oldwd <- getwd()
   setwd(root)
   on.exit(setwd(oldwd))
-
+print("now in writeImageJaspResults")
```
Comment (AlexanderLyNL) [inline JASP-R-Interface/jaspResults/R/writeImage.R]: Please remove the print statement.

Comment (vandenman) [reply JASP-R-Interface/jaspResults/R/writeImage.R]: good catch!

**Comment by JorisGoosen** [comment]
Well, the correlationplot shows up nicely now, even as eps. So when Alexander agrees I think we can merge this in?

I am working from home today, but we could discuss any open points here or through email? Or tomorrow.

**Comment by vandenman** [comment]
There are actually three additional fixes aside from the correlationplot:

- Saving the density plot crashed jasp (this was still a base plot). I changed it to Koen's ggplot code.
- Both the Bayesian anova and ancova didn't have a keep specified. In other words, JASP would immediately delete the .png files of the descriptives plots before they could be shown.

I'm happy discussing the code/ open points anywehere.

**Comment by JorisGoosen** [comment]
Well, this all looks fine to me so I've merged it in. 
The rest of the ggplots that @koenderks  will come in after the release of 0.9.1 right?

**Comment by koenderks** [comment]
Yes, I think that is what is supposed to happen.

**Pull request merged**